frontend/src/components/Forms/PartyRelationships/EditPartyRelationshipForm.js

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Field, reduxForm } from "redux-form";
+import { renderConfig } from "@/components/common/config";
+import { Form, Button, Col, Row, Popconfirm } from "antd";
+import * as FORM from "@/constants/forms";
+import { resetForm } from "@/utils/helpers";
+import EngineerOfRecordOptions from "@/components/Forms/PartyRelationships/EngineerOfRecordOptions";
+
+const propTypes = {
+  handleSubmit: PropTypes.func.isRequired,
+  closeModal: PropTypes.func.isRequired,
+  title: PropTypes.string.isRequired,
+  partyRelationship: PropTypes.object.isRequired,
+  mine: PropTypes.object.isRequired,
+  initialValues: PropTypes.object,
+};
+
+const defaultProps = {
+  mine: {},
+};
+
+const validate = (values) => {
+  const errors = {};
+  if (values.start_date && values.end_date) {
+    if (Date.parse(values.start_date) >= Date.parse(values.end_date)) {
+      errors.end_date = "Must be after start date.";
+    }
+  }
+  return errors;
+};
+
+export const EditPartyRelationshipForm = (props) => {
```

Comment:
Can `AddPartyRelationshipFrom` and `EditPartyRelationshipForm' be refactored to use the same component? 
If not, that's okay, just curious!

and Am I correct in assuming that `EngineerOfRecordOptions` was extracted out to be used in both places? I am not sure that the `Forms` file is the best place for it, maybe move it to `common` and keep `Forms` strictly for components that are Forms

Reply:
hmmm, I still think it belongs under the common directory, Most of the files under that directory as all children of forms as well.

Reply:
The components under common are common to the app, and where they are common to "forms" they are foundational components for all forms. 

These components will only be common to these specific two forms, and no others. Given that we may see 10+ of these child options, one for each contact type, this is going to clutter common with components that nobody else will care about.

Reply:
what about creating and adding them to `forms/common`? 
Assuming there will be 10+ this will clutter forms.

Reply:
To avoid clutter, I grouped them under a child folder called "PartyRelationships" along with the Add/Edit forms that relate to them. I can move them to forms/common, but I still have the same concern that they will apply only to these two forms and aren't really common re-usable components at the form level.

Reply:
ah no, that all works. 

Didn't notice the file grouping you had already created.

Reply:
Cool 👍